code/game/objects/items/weapons/autopsy.dm

```diff
@@ -167,21 +167,24 @@
 	if(!istype(M))
 		return 0
 
+	if (user.a_intent == I_HELP)
```

Comment:
"If they ARE on help intent, call the parent proc instead" == "If they AREN'T on help intent, actually perform the specific surgery applications of this tool

Afaik surgery is typically done explicitly on help intent, so is this intentional?

Reply:
If you are not on help, you will do the normal 'person is dead, do scan'. Otherwise, it will attempt the living surgery for scanning internal damage. Help is required because the parent checks for help.